SoftLayer/CLI/virt/detail.py

```diff
@@ -69,6 +69,10 @@ def cli(env, identifier, passwords=False, price=False):
     table.add_row(['transient', result.get('transientGuestFlag', False)])
     table.add_row(['created', result['createDate']])
     table.add_row(['modified', result['modifyDate']])
+    table.add_row(['preset', utils.lookup(result, 'billingItem',
+                                          'orderItem',
+                                          'preset',
+                                          'keyName') or {}])
```

Comment:
Use `-` as default to avoid printing a preset as `preset : {}  `
```suggestion
                                          'keyName') or '-'])
```